pyasic/data/boards.py

```diff
@@ -43,6 +45,8 @@ class HashBoard:
     expected_chips: int = None
     serial_number: str = None
     missing: bool = True
+    tuned: bool = True
```

Comment:
Maybe leave this to default to None, since I'm not sure how were gonna parse this for stock AM and even stock WM.